<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lab3_3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {background-color: rgb(200,200,200)}

QPushButton {color: black; background-color:rgb(123,183,1); border: 1px solid white; border-radius: 3px;}
QPushButton:hover { color: white; background-color:rgb(123,183,1); border: 3px solid white; border-radius: 3px;}

QLabel#maxLabel {background-color: white}
QLabel#minLabel {background-color: white}
QLabel#mediumLabel {background-color: white}
QLabel#drawingLabel {background-color: black}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fileLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>661</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Выбрать файл</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Нарисовать график</string>
    </property>
   </widget>
   <widget class="QLabel" name="file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Файл:</string>
    </property>
   </widget>
   <widget class="QLabel" name="drawingLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>780</width>
      <height>880</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>56</horstretch>
      <verstretch>86</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>780</width>
      <height>880</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>780</width>
      <height>880</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="moveXLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moveYLine">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="moveZLine">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rolateXLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rolateYLine">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rolateZLine">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="offsetX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Смещение по ОХ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="offsetY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Смещение по ОУ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="offsetZ">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Смещение по ОZ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rolateX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Поворот ОХ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rolateY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Поворот ОУ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rolateZ">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>320</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Поворот ОZ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Шаг по ОХ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scaleLineX">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="minimum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>630</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Минимум:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rangeMaxLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>670</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="rangeMinLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="maximum">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>630</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Максимум:</string>
    </property>
   </widget>
   <widget class="QLabel" name="range">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Диапазон значений</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Масштабирование</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scaleLineY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>520</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleY">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Шаг по ОУ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scaleZ">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Шаг по OZ:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="scaleLineZ">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="initButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>321</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Сбросить все параметры</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
